webui/react/src/pages/TrialDetails/TrialRangeHyperparameters.tsx

```diff
@@ -0,0 +1,210 @@
+import { Tooltip } from 'antd';
+import React, { useMemo } from 'react';
+
+import HumanReadableFloat from 'components/HumanReadableFloat';
+import {
+  ExperimentBase, ExperimentHyperParamType, TrialDetails,
+} from 'types';
+import { clamp } from 'utils/number';
+
+import css from './TrialRangeHyperparameters.module.scss';
+
+export interface Props {
+  experiment: ExperimentBase;
+  trial: TrialDetails;
+}
+
+interface HyperParameter {
+  name: string;
+  range: [number, number];
+  type: ExperimentHyperParamType;
+  val: string;
+  vals: string[];
+}
+
+const TrialRangeHyperparameters: React.FC<Props> = ({ experiment, trial }: Props) => {
+  const hyperparameters: HyperParameter[] = useMemo(() => {
+    const config = Object.entries(experiment.config.hyperparameters).map(([ name, value ]) => {
+      return { name, value };
+    });
+    const value = Object.entries(trial.hparams).map(([ name, value ]) => {
+      return { name, value };
+    });
+    return config.map(hp => {
+      return (
+        {
+          name: hp.name,
+          range: [ hp.value.minval || 0.00001, hp.value.maxval || 1 ],
+          type: hp.value.type,
+          val: String(value.find(ob => ob.name === hp.name)?.value || 0),
+          vals: hp.value.vals?.map(val => String(val)) ||
+          [ String(hp.value.minval || 0), String(hp.value.maxval || 1) ],
+        }
+      );
+    });
+  }, [ experiment.config.hyperparameters, trial.hparams ]);
+
+  return (
+    <div style={{ display: 'flex' }}>
```

Comment:
IMHO this looks better if wrapped in a `<Section bodyBorder>` (so there's a container with white background)

Reply:
and i think we need to make sure on mobile everything is visible, so either scrollable vertically or horizontally.